src/library/searchquery.cpp

```diff
@@ -12,8 +12,16 @@
 #include "util/db/sqllikewildcards.h"
 
 namespace {
-const QRegularExpression kDurationRegex(QStringLiteral("^(\\d*)(m|:)?([0-6]?\\d)?s?$"));
-const QRegularExpression kNumericOperatorRegex(QStringLiteral("^(>|>=|=|<|<=)(.*)$"));
+const QRegularExpression kDurationRegex(QStringLiteral("^(\\d+)(m|:)?([0-5]?\\d)?s?$"));
```

Comment:
Was the change from `0-6` to `0-5` intentional?

Reply:
Sure, :00-:59 is the valid range. I also added a test that checks that :60 is not accepted.

Reply:
The regex was wrong and no one ever noticed.